pyodide_build/tests/test_pywasmcross.py

```diff
@@ -208,8 +208,12 @@ def test_exports_node(tmp_path):
         """
     (tmp_path / "f1.c").write_text(template % (1, 1, 1))
     (tmp_path / "f2.c").write_text(template % (2, 2, 2))
-    subprocess.run(["emcc", "-c", tmp_path / "f1.c", "-o", tmp_path / "f1.o", "-fPIC"])
-    subprocess.run(["emcc", "-c", tmp_path / "f2.c", "-o", tmp_path / "f2.o", "-fPIC"])
+    subprocess.run(
+        ["emcc", "-c", tmp_path / "f1.c", "-o", tmp_path / "f1.o", "-fPIC"], check=False
+    )
+    subprocess.run(
+        ["emcc", "-c", tmp_path / "f2.c", "-o", tmp_path / "f2.o", "-fPIC"], check=False
+    )
```

Comment:
```suggestion
    subprocess.run(
        ["emcc", "-c", tmp_path / "f1.c", "-o", tmp_path / "f1.o", "-fPIC"], check=True
    )
    subprocess.run(
        ["emcc", "-c", tmp_path / "f2.c", "-o", tmp_path / "f2.o", "-fPIC"], check=True
    )
```

I think most places should have `check=True`, and especially for compilation commands where we don't collect the return code or stderr to use in the script later.